### ISSUE TITLE
Suggestion: Merge the person dots and the graph.

### ISSUE THREAD

**Issue by beruic** [GitFiend/Support#144]
For me the graph would be much easier to read if it was put on the left and merged with the person dots, so instead of each node in the graph, the there would be a circle with the users designation.

**Comment by beruic** [comment]
This way you can also make a dashed line from the head of the current branch to the local changes to graphically distinguish what is being worked on.

**Comment by GitFiend** [comment]
Yeah, I think the graph could be clearer about what lines up with what and the current branch.
